### ISSUE TITLE
[BUG] CcCrash (need help) 

### ISSUE THREAD

**Issue by Bexpg1337** [GrowtopiaNoobs/GrowtopiaServer#1387]
private void button3_Click_1(object sender, EventArgs e)
    {
        int netID = 0;
        World map = messageHandler.worldMap;
        foreach (Player p in map.players)
        {
            if (p.name.Contains(nameBoxOn.Text))
            {
                netID = p.netID;
                break;
            }
        }
        PacketSending.SendPacket((int)NetTypes.NetMessages.GENERIC_TEXT, "action|wrench\nnetid|" + netID.ToString(), realPeer);
        PacketSending.SendPacket((int)NetTypes.NetMessages.GENERIC_TEXT, "action|dialog_return\ndialog_name|popup\nnetID|" + netID.ToString() + "|\nbuttonClicked|" + actionButtonClicked.Text + "\n", realPeer);
    }

My server crashing with this bug how i can Fix ??
When i press in real gt its opening player dialog, and when 0 people in world and i press it show player left in world

**Comment by raiterjaki** [comment]
Because there is no player with that netid in the world.

**Comment by Bexpg1337** [comment]
**in real gt its opening player dialog, and when 0 people in world and i press it show player left in world**


"in my server auto crash"

**Comment by raiterjaki** [comment]
> **in real gt its opening player dialog, and when 0 people in world and i press it show player left in world**
> 
> "in my server auto crash"

Yeah because they made a thing that if a player tries to wrench a non-existing player it prevents it from getting crashed and it shows something else. But you didn't make anything about it and it can't get any data from the specified netid which probably causes the server to crash.

**Comment by Bexpg1337** [comment]
> > **in real gt its opening player dialog, and when 0 people in world and i press it show player left in world**
> > "in my server auto crash"
> 
> Yeah because they made a thing that if a player tries to wrench a non-existing player it prevents it from getting crashed and it shows that "player left the world". But you didn't make anything about it and it can't get any data from the specified netid which will probably cause the server to crash.

 i understand when i disabled wrench it dont crash anymore but i dont know how i can fix the error

**Comment by herothedev** [comment]
he literially just told ya

**Comment by Bexpg1337** [comment]
So how i can fix basically
